src/anjaliP/PrimeNumber.java

```diff
@@ -0,0 +1,22 @@
+package anjaliP;
+/*Assignment-22 : 18th Nov'2020 
+Print all prime numbers between 2 to 100.
+ */
+public class PrimeNumber {
+	
+	public void findPrimeNumber(int start,int end) {
+		System.out.println("Prime Numbers Between : "+start+" to "+end+" are -->");
+		for(int index=start;index<end;index++) {
+			boolean flag=true;
+			for(int innerIndex=2;innerIndex<Math.sqrt(index);innerIndex++) {
+				if(index%innerIndex==0) 
+					flag=false;
+			}
```

Comment:
Here, you should use break, because once you know this number is not prime then why we need to check that number with different vlaues